fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/util/EventNicenessPriorityRegister.java

```diff
@@ -46,4 +46,7 @@ public class EventNicenessPriorityRegister {
 
     // batteryUpdated
     public final static long BATTERY_UPDATED = -99_998_900;
+
+    // discover charging stations
+    public final static long CHARGING_STATIONS_DISCOVERY = -99_998_900;
```

Comment:
AFAIK, our `DefaultEventScheduler` doesn't care about the niceness of an event.
It uses a `PriorityQueue` using the time as priority. If there a ties, i.e. same execution time, Events are executed in an arbitrary order. I think ideally, we would use the defined niceness values to deterministically break those ties.

In the (highly unstable) `MultiThreadedEventScheduler` the niceness value is actually used. However, I'm not certain if it has any effect there.

So yes, we don't use this properly anyhow, but if we would it would be to break ties (I think)

@iwillitried for this reason the value of `CHARGING_STATIONS_DISCOVERY` should probably be unique.